### ISSUE TITLE
Span tag margin: change from top to bottom

### ISSUE THREAD

**Issue by dlewand691** [interactivethings/catalog#447]
When using the colors specimen, I noticed some excessive spacing between my heading and the specimen. 

Looking at the "span-span" class, I would recommend changing the margins from "24px 10px 0 0" to "0.5rem 2rem 0" so the margins are applied below and content on top will stay closer together. 

**BEFORE**
![before](https://user-images.githubusercontent.com/1159913/44478423-a7828500-a60b-11e8-857b-c3c8337bdb0f.jpg)

**AFTER**
![after](https://user-images.githubusercontent.com/1159913/44478424-a7828500-a60b-11e8-918a-851639a36a5d.jpg)

**Comment by herrstucki** [comment]
Are you overriding the heading margins? I think that might be the problem.

**Comment by dlewand691** [comment]
Yeah, overriding the headings but the span that's added between elements has a top margin that I'm having to remove.

**Comment by wereHamster** [comment]
This project has been deprecated and is no longer maintained.
